package.json

```diff
@@ -93,7 +95,9 @@
     "multer": "^1.4.1",
     "node-fetch": "^2.6.0",
     "nodemailer": "^6.2.1",
-    "winston": "^3.2.1"
+    "swagger-ui-express": "^4.0.7",
+    "winston": "^3.2.1",
+    "yamljs": "^0.3.0"
```

Comment:
A whole dependency just because we can't save an openapi spec as json :stuck_out_tongue:

Reply:
haha you can save open api spec as json, but I opted for yaml #YAML_LIFE

Reply:
as npm dependencies go, this one is actually not awful.